docs/quick-start/cmake.rst

```diff
@@ -4,7 +4,12 @@
 Notes about version of CMake
 ----------------------------
 
-* `3.5.0`_ **Minimum required**
+* `3.10.0`_ **Minimum required**
+
+  * CMake upstream dropping support for versions before 3.10
+  * `CMake release notes <https://cmake.org/cmake/help/latest/release/3.31.html#deprecated-and-removed-features>`__
```

Comment:
for completeness add a link to this PR
```suggestion
  * `CMake release notes <https://cmake.org/cmake/help/latest/release/3.31.html#deprecated-and-removed-features>`__
  * Since: `PR #769 <https://github.com/cpp-pm/hunter/pull/769>`__
```